files/kerberos_ktadd_key_helper_test.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+d=$(mktemp -d)
+k="$d"/t.keytab
+p1=a/b@C.D
+p2=e/f@C.D
+
+hwclock --systohc
+
+ktadd() {
+	date="$1"
+	kvno="$2"
+	date -s "$1"
+
+	cat << EOF | ktutil
+addent -password -p $p1 -k $kvno -e arcfour-hmac
+a
+addent -password -p $p2 -k $kvno -e arcfour-hmac
```

Comment:
We could just throw it out or somehow (I don't know how) convert it into spec unit tests.